include/cantera/oneD/StFlow.h

```diff
@@ -74,11 +74,15 @@ class StFlow : public Domain1D
     }
 
     //! set the transport manager
-    void setTransport(Transport& trans, bool withSoret = false);
+    void setTransport(Transport& trans, bool withSoret = false, bool withAmbi = false);
     void enableSoret(bool withSoret);
+	void enableAmbi(bool withAmbi);
     bool withSoret() const {
         return m_do_soret;
     }
+	bool withAmbi() const {
```

Comment:
Please configure your editor to indent code consistently with the rest of Cantera: 4 spaces per indentation level, and never use tabs.